torch/onnx/utils.py

```diff
@@ -729,6 +729,17 @@ def const_if_tensor(arg):
 # help the target of an ONNX export export more efficiently.  However,
 # ONNX doesn't currently formalize inplace. Fortunately, it's sound to drop
 # inplace annotations, but we are losing information this way.
```

Comment:
this comment looks unrelated to the below new function. Maybe add more blank lines in between